src/fetch/payouts.py

```diff
@@ -77,6 +77,7 @@ def __init__(  # pylint: disable=too-many-arguments
         solver: Address,
         solver_name: str,
         reward_target: Address,
+        bonding_pool: Address,
```

Comment:
Nit. This structure has become incredibly overloaded with field attributes. Might be time to consider splitting up the data structure into more logical components. 

Then you could remove the pylint disable above.